index.js

```diff
@@ -155,7 +156,7 @@ export function reactProcessor(opts = {}, components = {}) {
   [, opts] = setup('', opts);
   const { sanitize } = opts;
 
-  return htmlProcessor({ sanitize, ...opts })
+  return htmlProcessor(opts)
```

Comment:
I'm a bit nervous about changing the arg signature here, since I think we document this publicly and third parties may rely on it!

Reply:
I believe it's exactly the same. The prior code is copying the opts array for some reason?

Reply:
Yeah, I know. But I have a sneaking suspicion that that “some reason” might actually be important!